tests/nested/manual/refresh-revert-gadget-and-kernel/task.yaml

```
@@ -0,0 +1,83 @@
+summary: Refresh and revert the gadget and kernel snap
+
+description: |
+    This test validates the kernel and gadget snaps can be refreshed
+    and reverted to the new snaps published to edge channel.
+
+systems: [ubuntu-20.04-*]
+
+environment:
+    CORE_CHANNEL: beta
+    CORE_REFRESH_CHANNEL: edge
```

these aren't channels for the core snap, they are channels for the gadget or kernel snaps, so could we rename these to just SNAP_CHANNEL and SNAP_REFRESH_CHANNEL?

```suggestion
    SNAP_CHANNEL: beta
    SNAP_REFRESH_CHANNEL: edge
```

---

Thanks for hte review, in this case CORE_CHANNEL is already deifned in nested.sh and is the variable used to create the image, which was created when we just had core16. 
I agree that the name is not representative anymore, but as it requires many changes on other tests and spread.yaml I would prefer to make is in a folowing PR. 
Is it ok for you?

---

ah yes that's fine then I didn't realize that this var was needed/used in multiple places